add info on downgrading dbt with pip (#789)
fishtown-analytics/docs.getdbt.com: website/docs/docs/get-started/pip-install.md

All adapters build on top of `dbt-core`. Some also depend on other adapters: for example, `dbt-redshift` builds on top of `dbt-postgres`. In that case, you would see those adapters included by your specific installation, too.

### Upgrade adapters

To upgrade a specific adapter plugin:

```shell
pip install --upgrade dbt-<adapter>
```

> ### Install dbt Core only

Reviewer: ```suggestion
### Install dbt-core only
```

```shell
pip install dbt-core
```
### Upgrade or downgrade dbt Core versions

> To upgrade dbt to its latest version, use:

Reviewer: ```suggestion
You can upgrade or downgrade versions of dbt Core by using an upgrade command on the Command line (CLI). For more information, see [Best practices for upgrading in Core versions](/docs/dbt-versions/core#best-practices-for-upgrading).

To upgrade dbt to the latest version:
```

```shell
pip install dbt-core
```

> ### Upgrade or downgrade dbt Core versions

Reviewer 2: ```suggestion
### Change dbt Core versions
```

Reviewer 2: sugg for something shorter. maybe "Update" but that could be synonymous to upgrade for a lot of people.

non blocking.

> You can upgrade or downgrade versions of dbt Core by using an upgrade command on the Command line (CLI). For more information, see [Best practices for upgrading in Core versions](/docs/dbt-versions/core#best-practices-for-upgrading).

Reviewer 2: ```suggestion
You can upgrade or downgrade versions of dbt Core by using the `--upgrade` option on the command line (CLI). For more information, see [Best practices for upgrading in Core versions](/docs/dbt-versions/core#best-practices-for-upgrading).
```

To upgrade dbt to the latest version:

```
pip install --upgrade dbt-core
```

> To downgrade to an older version, you'll want to use the command above while specifying the version you want to use. As an example:

Reviewer 2: ```suggestion
To downgrade to an older version, specify the version you want to use. This command can be useful when you're resolving package dependencies. As an example:
```

```
pip install --upgrade dbt-core==0.19.0
```

> The `upgrade` flag is used to change versions of dbt, regardless of whether you're upgrading or downgrading via pip.

Reviewer 2: ```suggestion

```

Reviewer 2: sugg: i don't think we need this sentence anymore if my sugg for the first sentence in the first paragraph is accepted. 

reason: redundant

or, at least, remove the passive voice in the sentence
